src/pages/Products/Plugin.tsx

```
@@ -0,0 +1,161 @@
+                                    <div class="content-item-body">
+                                        <h4>{feature.title}</h4>
+                                        <p>{feature.description}</p>
+                                    </div>
+                                </div>
+                            )}
+                        </For>
+                        <div class="content-item-buttons">
+                            <a
+                                class="btn-primary"
+                                href="https://btcpay.docs.boltz.exchange/"
+                                target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)